projects/elife.yaml

```diff
@@ -154,6 +154,7 @@ defaults:
             bigquerylogging: false
             healthcheck: false
             errors: false
+            xpub: false
```

Comment:
not part of the generic schema of the `fastly` key, should go into a `vcl-variables` dictionary or similar that resides at this level

Reply:
Looking at the specific code later, evolves into:

```
vcl-templates:
    journal-submit: 
        xpub_uri: journal-submit
```
with a default of
```
vcl-templates: {}
```

Reply:
Yeah, wasn't sure about the best approach for this, sounds like a good solution.